src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.Linker.cs

```diff
@@ -2,50 +2,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.IO;
+using Mono.Cecil;
 
 namespace Xamarin.Android.Tasks
 {
 	public partial class MonoAndroidHelper
 	{
-		public static string [] TargetFrameworkDirectories;
-
-		internal static readonly string [] FrameworkEmbeddedJarLookupTargets = {
-			"Mono.Android.Support.v13.dll",
-			"Mono.Android.Support.v4.dll",
-			"Xamarin.Android.NUnitLite.dll", // AndroidResources
-		};
-		internal static readonly string [] FrameworkEmbeddedNativeLibraryAssemblies = {
-			"Mono.Data.Sqlite.dll",
-			"Mono.Posix.dll",
+		static readonly HashSet<string> KnownAssemblyNames = new (StringComparer.Ordinal) {
```

Comment:
Use of `.Ordinal` here also concerns me, as `IsFrameworkAssembly(string)` + `Path.GetFileNameWithoutExtension()` assumes that the filesystem -- and all tooling which puts file on disk -- doesn't "lose" case and/or *consistently* uses the same case everywhere, and I'm not convinced that's true.  See also our history around `Resource.Designer.cs` vs. `Resource.designer.cs` and #5743.

I think using `.OrdinalIgnoreCase` may be safer.